src/components/mainPage/index.jsx

```diff
@@ -1,24 +1,20 @@
-import React from "react";
-import { useHistory } from "react-router-dom";
-import { ExportOutlined } from '@ant-design/icons';
-import { logoutUser } from "../../services/authentication";
+import Header from "../navigation/header";
+import { GetUserName } from "../../services/userService";
 
 function MainPage() {
-
-    let history = useHistory();
-
-    const logOut = () => {
-        logoutUser(history);
-    };
+    GetUserName();
```

Comment:
Why it hangs in the air and is not called at the start of the page ? It is called three times.

![Screenshot 2022-05-04 211802](https://user-images.githubusercontent.com/56841893/166800205-17804c93-adb7-4112-9879-4c61350ee894.jpg)